osu.Game.Rulesets.Osu/Skinning/Legacy/LegacyCursorParticles.cs

```diff
@@ -248,6 +347,23 @@ private enum SpewDirection
             Left,
             Right,
             Omni,
+            RelaxHit,
+        }
+
+        private partial class LegacyRelaxCursorParticleSpewer : LegacyCursorParticleSpewer
+        {
+            // Disable spawn in Update()
+            protected override bool CanSpawnParticles => SpawnKiaiParticles;
+
+            public LegacyRelaxCursorParticleSpewer(Texture? texture, int perSecond)
+                : base(texture, perSecond)
+            {
+                SpawnKiaiParticles = false;
+            }
+
+            public void SpawnNewParticles() => SpawnParticle();
```

Comment:
Why is this named `SpawnNewParticles()` (plural) if it spawns just one? And why is this an actual method, which is next to `SpawnKiaiParticles` and also `CanSpawnParticles` (which this one method doesn't even check)?

Reply:
`SpawnKiaiParticles` is used for the effect I said below, while `CanSpawnParticles` is used for controlling spawn particles in `Update()`. Considering `Update()` spawns constantly if `Active` and `CanSpawnParticles` were true, it's difficult to make hit particles spawn this way. So I exposed `spawnParticle()` to do so.

My previous code may be more reasonable.
``` c#
            public void SpawnRelaxHitParticles()
            {
                if (!base.CanSpawnParticles) return;

                for (int i = 0; i < 6; i++)
                    SpawnParticle();
            }
```